### PR TITLE
Breaking changes in index.js

### PR DESCRIPTION
**Pull request by josiasbudaydeveloper**
# Now the chatbot will always answer the questions separately
## Explanation
The previous intention was build something like ChatGPT, where it can understand the changes in the context during the conversasion. 

This is a complex feature and may be it will be implemented in the future, but for now, I will make all questions be treated separetely.

In addition, another disadvantage is that every new question will send the entire information about CloudWalk, so the costs will be a little higher, because the API charges $0.002/token.

## In summary, these are the disadvantages of this solution:
- The chat do not understand any information between the questions
- The costs related to the API behind the chat will be a little higher, because the API charges $0.002/token and the messages of the question will always be with the entire information about CloudWalk.

### CODE REVIEW

**Comment by josiasbudaydeveloper** [comment]
#1 Now we just need to update the README.md
